site/docs/4.4/components/forms.md

```diff
@@ -1003,12 +970,18 @@ We recommend using client-side validation, but in case you require server-side v
 
 Validation styles are available for the following form controls and components:
 
-- `<input>`s and `<textarea>`s with `.form-control` (including up to one `.form-control` in input groups)
+- `<input>`s and `<textarea>`s with `.form-control`
 - `<select>`s with `.form-control` or `.custom-select`
 - `.form-check`s
 - `.custom-checkbox`s and `.custom-radio`s
 - `.custom-file`
 
+{% capture callout %}
+##### Input group validations
+For input groups, put feedback messages as a sibling element of the input group and add the `.is-{valid|invalid}` class to the `.input-group`. Placing feedback messages in a input group encounters border radius issues. [See this comment for workaround](https://github.com/twbs/bootstrap/issues/25110#issuecomment-586565165).
```

Comment:
This needs word proofing. Also, I'm not sure I like linking to a comment, because the comment can be edited/removed...